src/core/package.ts

```diff
@@ -0,0 +1,46 @@
+import Table from 'cli-table3';
+import chalk from 'chalk';
+import FieldLoader from '../util/fieldLoader';
+import Differ from '../util/differ';
+
+class Package {
+  fieldLoader = new FieldLoader();
+  differ = new Differ();
+
+  async runLocalCheck(): Promise<void> {
+    const requiredFields = await this.fieldLoader.loadFields(null);
+    const packageFields = await this.fieldLoader.loadFields('./package.json');
+    // get report from differ
+    const report = this.differ.run(requiredFields, packageFields);
+
+    // print report
```

Comment:
Not tested yet but maybe the code of creating the table can be extracted to another method in this class.
But that is something we can take a look later

Reply:
I agree with this one also. I just built it like that for the time being.